src/graph/validator/MatchValidator.cpp

```diff
@@ -45,7 +45,10 @@ Status MatchValidator::validateImpl() {
 
         auto matchClauseCtx = getContext<MatchClauseContext>();
         matchClauseCtx->aliasesUsed = aliasesUsed;
-        NG_RETURN_IF_ERROR(validatePath(matchClause->path(), *matchClauseCtx));
+        if (matchClause->path()->pathSize() > 1) {
+          return Status::SemanticError("Multi paths not supported.");
+        }
```

Comment:
The data structure of `MatchClauseContext` may need to be redesigned.

Reply:
Will do.